.github/workflows/ci.yaml

```diff
@@ -14,45 +14,13 @@ jobs:
         run: python3 -m pip install tox
       - name: Run linters
         run: tox -e lint
-
-  unit-test:
-    name: Unit tests
-    needs:
-      - lint
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Install dependencies
-        run: python -m pip install tox
       - name: Run tests
         run: tox -e unit
 
-  integration-test-lxd-charm:
```

Comment:
This was also an experiment to lower the footprint of an average test run, especially when we're running enough CI that it's becoming a problem for other developers in the org. I think testing the charm enters a blocked state before applying TLS could be carried out during the TLS tests; considering this charm is supposed to not function without TLS, I think we could refactor thse tests to combine normal charm tests (such as this blocked test) with TLS tests, since they're both part of our expected baseline functionality.